website/gatsby-config.js

```diff
@@ -35,7 +35,7 @@ module.exports = {
     },
   },
   plugins: [
-    `gatsby-plugin-ts`,
+    `gatsby-plugin-graphql-codegen`,
```

Comment:
This plugin produces the following warning: `warn [gatsby-plugin-graphql-codegen] Identifier 'getIterator' has already been declared (3564:9)`

Reply:
I will resolve this one. But let's try targeting zero warning builds 😄